cromwell-drs-localizer/src/main/scala/drs/localizer/downloaders/GetmChecksum.scala

```diff
@@ -26,23 +25,18 @@ sealed trait GetmChecksum {
 }
 
 case class Md5(override val rawValue: String) extends GetmChecksum {
-  override def value: ErrorOr[String] = {
-    val trimmed = rawValue.trim
-    if (trimmed.matches("[A-Fa-f0-9]+"))
-      trimmed.validNel
-    // TDR currently returns a base64-encoded MD5 because that's what Azure seems to do. However,
-    // the DRS spec does not specify that any checksums should be base64-encoded, and `getm` also
-    // does not expect base64. This case handles the current behavior in the short term until
-    // https://broadworkbench.atlassian.net/browse/DR-2259 is done.
```

Comment:
DR-2259 has been done for a while. Confirmed with a TDR engineer (@okotsopoulos) that we should be getting hex strings for all md5 hashes now.